test/unit-test/cellular_pktio_utest.c

```diff
@@ -1679,7 +1676,7 @@ void test__Cellular_PktioSendAtCmd_Invalid_String( void )
     CellularPktStatus_t pktStatus = CELLULAR_PKT_STATUS_OK;
     CellularContext_t context;
     CellularCommInterface_t * pCommIntf = &CellularCommInterface;
-    CellularCommInterfaceHandle_t commInterfaceHandle;
+    CellularCommInterfaceHandle_t commInterfaceHandle = ( CellularCommInterfaceHandle_t ) malloc( sizeof( CellularCommInterfaceHandle_t ) );
```

Comment:
malloc fails?

Reply:
please refer to this [change](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Library/pull/37/commits/d6ed9baaca207edc8cb5ab7fe890e2937975d2d8).